package.json

```diff
@@ -56,6 +65,7 @@
     }
   },
   "pre-commit": [
+    "lint",
```

Comment:
We should add `lint` as part of test - that way it'll run in Travis CI, as well (and then remove it from the pre-commit list).